README.md

```diff
@@ -1,2 +1,78 @@
 # rust
-Docker image for Rust
+
+This is a Docker image with Rust 1.18 installed.
+
+## How to Use
+
+In order to use this image for your project, edit the `techio.yml` file and add the following lines to your project:
+
+```yaml
+    runner: techio/rust:latest
+```
```

Comment:
So, sounds good to add a `buildCommand` in the `techio.yml` to launch the `cargo build`? In this case, should be done in the template if we use this rust image.